corehq/apps/app_manager/templates/app_manager/v2/form_view_base.html

```diff
@@ -211,7 +211,7 @@
 
         gaTrackLink($('.breadcrumb .view-form'), 'App Builder', 'View Form', 'Breadcrumb');
 
-        {% if allow_cloudcare %}
+        {% if can_preview_form and allow_cloudcare %}
```

Comment:
Also what Simon said makes sense. Also stated by @benrudolph [here](http://manage.dimagi.com/default.asp?244649#bugevent_1268745). So added changes for Preview App as well and made the corresponding change in [commit](https://github.com/dimagi/commcare-hq/pull/14457/commits/4876d3fd5b5c1a5421b09d86f737c4401a8ad4b6)